WilliamsGarcia/30-08/HackerRank/Map/MapExample.java

```diff
@@ -0,0 +1,35 @@
+package com.bootcamp.daythree.hackerrank.map;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class MapExample {
+
+    public static final String NOT_FOUND = "Not found";
+    public static final String SEPARATOR = " = ";
+
+    public static void main(String[] args) {
+
+        Scanner scan = new Scanner(System.in);
+        Map<String, Integer> contact = new HashMap<>();
```

Comment:
When you variable is a Collection or Map, it is good practice put name in singular and them the the type, example: tableListList, userSet, etc...